### ISSUE TITLE
command 'edit-csv.edit' not found

### ISSUE THREAD

**Issue by Juku7** [janisdd/vscode-edit-csv#150]
vscode on windows 10
Whenever I click on the Edit CSV button on the top right, I got this error message " command 'edit-csv.edit' not found ". It's used to work fine without any problem. I don't know why this happened! Also, rainbow CSV extension is not working too 
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/0d4257c4-b401-4b74-9184-91777e94210a)

Im trying to uninstall the extension and even reinstall vscode but those did not help either.

**Comment by janisdd** [comment]
What does vs code shows as "Language" in the bottom right corner? It should say something with "csv" and not "Plain Text".
Try searching for "CSV: edit csv" in the command palette. Does this work?

**Comment by Juku7** [comment]
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/45b4f91c-23ec-4287-8208-ae5f2e48a66a)

vscode detects the file as csv but the extension seems to be not.
Using command palette > CSV: edit csv still got an error as screenshot above.

**Comment by janisdd** [comment]
Since rainbow csv  doesn't work either, it seems that settings is vs code are wrong. 
Are other extensions working as expected? I haven't worked much with settings profiles, but maybe you can export all settings, reset all settings and see if it works with all settings at default values. Then you can import the old settings.

This doesn't seem to be specifically related to this extension.

**Comment by Juku7** [comment]
Thank you! I'll try your suggestion and the other extensions working as expected.

**Comment by Juku7** [comment]
I have tried resetting user setting and everything back to default but still got the same error with Edit CSV and Rainbow CSV.
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/63b2ead5-b022-460e-ab96-d710831b23e4)

**Comment by janisdd** [comment]
This is really strange... You could also try opening the development tools `Developer: Toggle Developer Tools` via the command palette and look for error messages. What are the messages when you click on the "edit csv" button?

Maybe execute the palette command `Extensions: Open Extensions Folder` and check if its the correct one according to https://code.visualstudio.com/docs/editor/extension-marketplace#_common-questions ?

**Comment by Juku7** [comment]
These are the error messages that I get in the developer tools
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/c8711724-656a-4f85-a689-d4ef3d6ffd39)
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/3ed60e95-6cfb-4981-9944-9311614c2047)
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/edcef5d0-6413-483a-8459-6c615d621037)

and extension folder is in the correct place:
![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/57dfba27-cd90-474c-9ae1-e97e5f812a0b)

**Comment by janisdd** [comment]
I've found this: https://github.com/microsoft/vscode/issues/136537 The problem might be related to another extensions that prevents other extensions from working. 
They suggest to run the command `Help: Start Extension Bisect`. I haven't used this feature myself so I can't tell you how it works directly. Alternatively, you can disable all extensions and re-enable them one by one and check if csv works.

**Comment by Juku7** [comment]
I followed your last suggestion and everything works now! Thank you very much!
So I uninstalled the extensions one by one, but only 3 of them, the first Rainbow CSV, the second CSV editor and the third Excel viewer. I feel that Excel Viewer is causing problems because things do not work, expect Excel Viewer when there is a problem.

After restarting vscode, I reinstalled Rainbow CSV and it worked perfectly, even the Edit CSV. I also reinstalled Excel Viewer and it's also working fine. 

![image](https://github.com/janisdd/vscode-edit-csv/assets/134831736/b8168560-7303-4fd5-b528-1149a5563d7c)
Again thank you so much for your help!!
